src/components/NewCommentForm.tsx

```diff
@@ -175,7 +176,7 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
           <button
             type="submit"
             className={classNames('button', 'is-link', {
-              'is-loading': submitting,
+              'is-loading': loaded,
```

Comment:
The 'is-loading' class should still depend on a local 'submitting' state, not on the 'loaded' state from the store. Update the useState and logic for handling 'submitting' state.